<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Neural</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Текущая нейроная сеть</string>
    </property>
    <widget class="QLabel" name="lName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>401</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Название:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lCountTraining">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>401</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Количество примеров обучения сети:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lValueTraining">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>401</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Коэффицент обучения:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lCountInputNeuron">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>401</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Количество нейронов входного слоя</string>
     </property>
    </widget>
    <widget class="QLabel" name="lCountOutputNeuron">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>401</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Количество нейронов выходного слоя</string>
     </property>
    </widget>
    <widget class="QLabel" name="lCountHiddenNeuron">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>401</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Количество нейронов промежуточного слоя</string>
     </property>
    </widget>
    <widget class="QLabel" name="lCountHiddenNeural">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>401</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Количество промежуточных слоев</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>401</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="verticalSpacing">
       <number>7</number>
      </property>
      <item row="1" column="0">
       <widget class="QPushButton" name="bSaveNeural">
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="bSaveAsNeural">
        <property name="text">
         <string>Сохранить как</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="bTestNeural">
        <property name="text">
         <string>Обучение и тестирование</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>449</x>
      <y>9</y>
      <width>241</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Изменить текущую нейроную сеть</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>221</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QPushButton" name="bOpenNeural">
        <property name="text">
         <string>Открыть существующую</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="bCreateNewNeural">
        <property name="text">
         <string>Создать новую</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
